packages/tldraw/src/state/commands/shared/removeShapesFromPage.ts

```diff
@@ -1,5 +1,14 @@
 import { TLDR } from '~state/TLDR'
-import type { ArrowShape, TDSnapshot, GroupShape, PagePartial } from '~types'
+import type { ArrowShape, GroupShape, PagePartial, TDPage, TDSnapshot } from '~types'
+
+const doesAssetExistInAnotherShape = (page: TDPage, deletedShapes: string[], assetId: string) =>
```

Comment:
Assets are shared between pages! I'm going to update this logic with a broader sweep at the end of the function.

Reply:
🙌 Thanks @steveruizok